gateway/operations/deleteobject.go

```diff
@@ -51,7 +52,7 @@ func (controller *DeleteObject) Handle(w http.ResponseWriter, req *http.Request,
 	lg := o.Log(req).WithField("key", o.Path)
 	err := o.Cataloger.DeleteEntry(req.Context(), o.Repository.Name, o.Reference, o.Path)
 	switch {
-	case errors.Is(err, db.ErrNotFound):
```

Comment:
Graveler now won't return ErrNotFound on delete, so I guess you don't need this

Reply:
Agreed.  It's just too close for comfort.

I look forward to deleting `db.ErrNotFound` when we remove Postgres.